### PR TITLE
Jc fixed mistake

### PR DESCRIPTION
**Pull request by jaycastroram**

### CODE REVIEW

**Review by kennypanyavong — COMMENTED** [review]
everything looks good in the browser

File: scripts/services.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const ParkService = () => {
     let html = '';
     
     for (const serv of services) {
-        html += `<li data-type="service" data-serviceid="${serv.id}" data-name="${serv.name}">${serv.name}</li>`;
+        html += `<lu class="services" data-type="service" data-serviceid="${serv.id}" data-name="${serv.name}">${serv.name}, </lu>`;
```
Comment (kennypanyavong) [inline scripts/services.js:39]: just caught this, correct typo lu to ul

**Review by kennypanyavong — CHANGES_REQUESTED** [review]

File: scripts/services.js (hunk 1 above)
Comment (jaycastroram) [reply scripts/services.js]: issue was adjusted

**Review by kennypanyavong — COMMENTED** [review]
good to go

**Pull request merged**